pkg/awsutils/awsutils.go

```diff
@@ -396,6 +397,7 @@ func New(useSubnetDiscovery, useCustomNetworking, disableLeakedENICleanup, v4Ena
 	log.Infof("Subnet discovery enabled %v", cache.useSubnetDiscovery)
 	cache.v4Enabled = v4Enabled
 	cache.v6Enabled = v6Enabled
+	cache.disableENIMetadataReconcile = disableENIMetadataReconcile
```

Comment:
I guess same question here, should we avoid adding to 'cache' obj. Looks like we just need to read the input param and skip the section in DescribeAllENIs()